src/components/ConfirmationPage.tsx

```diff
@@ -31,6 +34,9 @@ type ConfirmationPageProps = {
     /** Additional style for the heading */
     headingStyle?: TextStyle;
 
+    /** Additional style for the animation */
+    animationStyle?: ViewStyle;
```

Comment:
If you change the name of 'animation' prop, then remember to also change this name.